pkg/agent/testdata/AKSUbuntu1804+Containerd+Kubenet/line200.sh

```diff
@@ -1,9 +0,0 @@
-[Unit]
```

Comment:
What are all these `line200.sh` changes?

Reply:
We have a bunch of pre-defined node provisioning scenarios (i.e containerd + kubenet cni) and our make generate target runs templating engine to generate the cloudinit files - this is mostly just a way to sanity check that the outputs are correct
not 100% sure the output files are named like that, tbh

Reply:
for the most part it is `s/io.containerd.runtime.v1.linux/io.containerd.runc.v2/` apart from the generated test data like this file?
Not sure if I need to specify default_runtime but it seems like it's better to be safe?

the nvidia-gpu-runtime configs need a bit more lines of config to specify the binary location  so I've completely separate the templating to make it more readable